microk8s-resources/wrappers/addon-lists.yaml

```diff
@@ -159,3 +159,10 @@ microk8s-addons:
       supported_architectures:
       - amd64
       - arm64
+
+    - name: "ambassador"
+      description: "Ambassador API Gateway and Ingress"
+      version: "1.5.1"
```

Comment:
Is version accurate? I see we get the latest manifest.

Reply:
Yes, in fact it is installing the latest... I've seen some other addons leave this field empty: I will do that if it is ok or you.